policyengine_us/parameters/gov/states/me/tax/income/deductions/personal_exemption/phaseout/start.yaml

```diff
@@ -1,21 +1,43 @@
 description: Maine starts phasing out its personal exemption for filers with Maine AGI above this threshold.
 metadata:
   unit: currency-USD
+  period: year
+  uprating:
+    parameter: gov.irs.uprating
+    rounding:
+      type: downwards
+      interval: 50
   label: Maine personal exemption phase-out start
   reference:
-    - title: Instructions for Form 1040ME
+    - title: 2023 Instructions for Form 1040ME Line 2
+      href: https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/23_1040me_book_gen_instr.pdf#page=4
+    - title: §5403. Annual adjustments for inflation 8
+      href: https://legislature.maine.gov/statutes/36/title36sec5403.html
+    - title: 2022 Instructions for Form 1040ME Line 18 Exemption
+      href: https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/22_1040me_book_gen_instruc_revisedFeb23.pdf#page=4
+    - title: 2021 Instructions for Form 1040ME
       href: https://www.maine.gov/revenue/sites/maine.gov.revenue/files/inline-files/21_1040me_book_gen_instr_revMay2022.pdf#page=4
     - title: §5126-A. Personal exemptions on or after January 1, 2018
       href: https://legislature.maine.gov/statutes/36/title36sec5126-A.html
   breakdown:
     - filing_status
 SINGLE:
```

Comment:
We need to add the uprating to each component of the breakdown file:

See example
![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/990531a0-9c27-4f33-9da7-89ac6ac81b7c)

Reply:
all breakdown files

Reply:
> all breakdown files

Not sure how to add values for 'amount' in 'property_tax'.
![15b9f1c006c47014c307b34c0872a19](https://github.com/PolicyEngine/policyengine-us/assets/98068600/381d25ff-a28c-4c46-ab65-add3f792b60e)